Problem1.py

```diff
@@ -0,0 +1,21 @@
+'''
+Find missing element in sorted array in range 1 to n
+Time complexity: O(logn)
+Space complexity: O(1)
+'''
+class Solution:
+    def findMissingElement(self, arr):
+        low = 0
+        high = len(arr) - 1
+        while low < high:
+            mid = low + (high - low)/2
```

Comment:
Great work on getting the TC and SC right. Please mention the approach in your comments as well.

Also, `mid = low + (high - low)/2` would give you a floating point number.